tests/e2e/lib.sh

```diff
@@ -322,6 +322,11 @@ deploy_sensor() {
         # et al.
         kubectl -n "${sensor_namespace}" set resources deploy/sensor -c sensor --requests 'cpu=2' --limits 'cpu=4'
     fi
+    # Bump Sensor's memory to avoid OOMing.
```

Comment:
What are the default values for memory request/limits with operator deployments? It would be useful not to deviate from the default customer installs.

Reply:
I've bumped the values to match the operator's. I've also removed the comment indicating that this should be reverted because on second thought I think we should keep these values.